ci/deploy_app.sh

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env bash
+set -exo pipefail
+
+REGION="${REGION:=europe-west2}"
+
+CONCURRENCY="${CONCURRENCY:=80}"
+MIN_INSTANCES="${MIN_INSTANCES:=1}"
+MAX_INSTANCES="${MAX_INSTANCES:=1}"
+CPU="${CPU:=4}"
+MEMORY="${MEMORY:=4G}"
+
+WEB_SERVER_TYPE="${WEB_SERVER_TYPE:=gunicorn-threads}"
+WEB_SERVER_WORKERS="${WEB_SERVER_WORKERS:=7}"
+WEB_SERVER_THREADS="${WEB_SERVER_THREADS:=10}"
+WEB_SERVER_UWSGI_ASYNC_CORES="${WEB_SERVER_UWSGI_ASYNC_CORES:=10}"
+DATASTORE_USE_GRPC="${DATASTORE_USE_GRPC:=True}"
```

Comment:
This is a runner environment variable, so should be in the next group of variables.

Reply:
Moved, thanks!